gui/frontend/src/search/search.coffee

```diff
@@ -368,7 +373,20 @@ angular.module "mindbender.search", [
             @types = null
             @indexes = null
             @elastic = elasticsearch
-            @collapsed_facets = {}
+            @collapsed_facets = { 
```

Comment:
See Dev instance. Some are collapsed, some not.

On Tuesday, November 10, 2015, alldefector <notifications@github.com> wrote:

> In gui/frontend/src/search/search.coffee
> <https://github.com/HazyResearch/mindbender/pull/63#discussion_r44487765>:
>
> > @@ -368,7 +373,20 @@ angular.module "mindbender.search", [
> >              @types = null
> >              @indexes = null
> >              @elastic = elasticsearch
> > -            @collapsed_facets = {}
> > +            @collapsed_facets = {
>
> are you collapsing all of them upon load?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HazyResearch/mindbender/pull/63/files#r44487765>.
>